services/121-service/src/registration/validators/any-valid-body.validator.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  createParamDecorator,
+  ExecutionContext,
+  ValidationPipe,
+} from '@nestjs/common';
+
+const RawBody = createParamDecorator(
+  (_data: unknown, ctx: ExecutionContext): any => {
+    const request = ctx.switchToHttp().getRequest();
+    return request.body;
+  },
+);
+
+// This custom decorator is needed to allow external callbacks with unknown or extra properties
+// It applies a ValidationPipe with relaxed settings, so the global whitelist does not block valid payloads
```

Comment:
whiteList = true both here and globally. So what is the difference? In the previous version we were passing whiteList: false to the exception endpoints.